### DISCUSSION TITLE
Add Countdown Timer To Next Live Session

### DISCUSSION THREAD

**Discussion by Slooki** [f1multiviewer/issue-tracker#137] [Feature requests]
Something like they have on F1 website. Thanks 💚

**Comment by Slooki** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/35727372/181914187-c8340a85-01a2-4eda-b022-f177ad64929a.png)
Without sponsors haha

**Comment by f1multiviewer** [comment]
https://github.com/f1multiviewer/issue-tracker/discussions/248 merged into this one

**Reply by LapsTimeOFF** [reply]
You mean #248 ?

**Reply by f1multiviewer** [reply]
Yes you're right!
